app/controllers/staff/speakers_controller.rb

```diff
@@ -70,7 +68,12 @@ def speaker_params
     params.require(:speaker).permit(:bio, :email, :user_id, :event_id, :speaker_name, :speaker_email)
   end
 
-  def set_proposal
-    @proposal = Proposal.find_by(uuid: params[:proposal_uuid])
+  def set_program_session
+    @program_session = current_event.program_sessions.find_by(id: params[:program_session_id])
+  end
+
+  def speaker_count_check
+    @speaker = current_event.speakers.find(params[:id])
+    @speaker.program_session.speakers.count > 1
   end
```

Comment:
If the count is > 1, display the appropriate flash message and redirect back to the page.